common/types/activation.go

```diff
@@ -655,8 +655,11 @@ type PostMetadata struct {
 	Challenge     []byte
 	BitsPerLabel  uint8
 	LabelsPerUnit uint64
-	K1            uint32
-	K2            uint32
+
+	K1 uint32
+	K2 uint32
+	B  uint32
+	N  uint32
```

Comment:
would be nice to add some documentation, or point to a doc, to help readers understand these parameters.

Reply:
These values are documented in the PoST repository here: https://github.com/spacemeshos/post/blob/b36de2e6292c3247b359667b4d563bb4c0cd7562/config/config.go#L55-L59